packages/web/src/components/spa/HackAUBG/SponsorsSection/sponsors_section.css

```diff
@@ -23,7 +23,56 @@
     font-weight: 900;
     font-size: 2em;
 }
-/* Optimize the following css by grouping all containers that use the same code */
+
+.sponsors-box-header-container-platinum {
```

Comment:
I don't know why you changed the CSS classes for .sponsors-box-header-container-(category). They used to be grouped because they apply most of the same values for the properties. The only ones that I wrote separately were for border-color and background-color because they're different for each category. That way you can save space and not write almost the same class 5 separate times. The way it is now, it looks very unoptimized.